test/system/static.test.ts

```diff
@@ -495,3 +495,26 @@ test("static and dynamic scanning results are aligned", async (t) => {
   );
   // TODO: imageLayers is completely different
 });
+
+test("static scanning with dockerfile analysis matches expected values", async (t) => {
+  const thisIsJustAnImageIdentifierInStaticAnalysis = "nginx:latest";
+  const dockerfilePath = path.join(__dirname, "../fixtures/dockerfiles/library/nginx/Dockerfile");
+  const pluginOptionsWithDockerSave = {
+    staticAnalysisOptions: {
+      imagePath: getFixture("docker-save/nginx.tar"),
+      imageType: ImageType.DockerArchive,
+    },
+  };
+
+  const pluginResultStatic = await plugin.inspect(
+    thisIsJustAnImageIdentifierInStaticAnalysis,
+    dockerfilePath,
+    pluginOptionsWithDockerSave,
+  )
+
+  t.equals(
+    pluginResultStatic.scannedProjects[0].depTree.docker.baseImage,
+    "debian:stretch-slim",
+    "base image matches expected",
+  );
+});
```

Comment:
newline 😛

Reply:
BAH. I've configured my vs-code to fix this automatically from now on so shouldn't be an issue in the future.